runtime/src/test/scala/akka/grpc/akka/grpc/scaladsl/GrpcExceptionHandlerSpec.scala

```diff
@@ -0,0 +1,71 @@
+package akka.grpc.akka.grpc.scaladsl
```

Comment:
This package name is rather strange - that seems unintentional? (and causes compilation problems, see CI)

Reply:
Fixed